function throw(n){throw new Error(n);}